pegasus/sites.v3/code.org/views/featured_tutorial.html

```diff
@@ -2,7 +2,7 @@
   <a href="%url%" style="font-family: 'Gotham 4r', sans-serif; color: black; text-decoration: none">
     <div style="background: white; border: 1px solid rgb(187, 187, 187); border-radius: 10px; height: 290px; overflow: hidden; text-align: left">
       <div style="width: 100%; height: 120px; overflow: hidden">
-        <img src="%image%" style="width: 100%"/>
+        <img src="%image%" style="width: 100%" alt=""/>
```

Comment:
Added empty alt text here since these images are decorative. This is used on https://code.org/hourofcode2022 too.